src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -379,7 +399,18 @@ static ConfigDef newConfigDef() {
             REBALANCING_DEFAULT,
             Importance.LOW,
             "Whether to trigger a rebalancing by exceeding the max poll interval (Used only in"
-                + " testing)");
+                + " testing)")
+        .define(
+            INGESTION_METHOD_OPT,
+            Type.STRING,
+            INGESTION_METHOD_DEFAULT_SNOWPIPE,
+            IngestionMethodConfig.VALIDATOR,
+            Importance.LOW,
+            "Use 1.0 or 2.0 for Snowpipe or Streaming ingest respectively",
```

Comment:
Since this is public facing should we say something like "Use 1.0 for Snowpipe file based ingestion or 2.0 for Snowpipe streaming?

Reply:
Yes, let me reword it. I removed the 1.0 and 2.0 since it might be confusing.

Reply:
Instead, accepted values are SNOWPIPE and SNOWPIPE_STREAMING